src/components/dnd-table-v2/dnd-table-bottom-left-buttons.tsx

```
@@ -5,7 +5,8 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-import { Grid, IconButton, Tooltip } from '@mui/material';
+import { Grid, IconButton } from '@mui/material';
+import { CustomTooltip } from '../tooltip/CustomTooltip';
```

ESLint

[error] `../tooltip/CustomTooltip` import should occur after import of `react-intl`

(import-x/order)